pkg/repo/badgerdb/service.go

```diff
@@ -0,0 +1,132 @@
+package badgerdb
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/dgraph-io/badger/v3"
+	"github.com/google/uuid"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo/common"
+)
+
+const servicePrefix = "service:"
+
+// getServiceKey converts a service ID to a BadgerDB key with prefix.
+func getServiceKey(id uuid.UUID) []byte {
+	return []byte(fmt.Sprintf("%s%s", servicePrefix, id))
+}
+
+// getValue unmarshals the value of a BadgerDB item into v (either *model.Service or *model.Messages).
```

Comment:
Move this function to `badgersdb.go` so we have common functionality of the package in one place.